src/pages/CallBack.tsx

```diff
@@ -33,7 +33,7 @@ export default function CallBackPage() {
             }
         }
 
-        const redirectJwtPayload = redirectJwt.payload;
+        const redirectJwtPayload = redirectJwt.getPayload();
         const url =
```

Comment:
follow up is we need to set storage to have the account name and DID of the account, which should also be checked when the user opens the site to see if they are already logged in. New Task

Reply:
agreed